src/imitation/scripts/expert_demos.py

```diff
@@ -44,14 +41,15 @@ def rollouts_and_policy(
   rollout_save_n_timesteps: Optional[int] = None,
   rollout_save_n_episodes: Optional[int] = None,
 
-  ray_tune_interval: int = 1,
+  ray_tune_reporter=None,
+  ray_tune_interval: int = -1,
 
   policy_save_interval: int = -1,
   policy_save_final: bool = True,
 ) -> dict:
   """Trains an expert policy from scratch and saves the rollouts and policy.
 
-  Ray Tune (turn on using `ray_tune_interval > 0`):
+  Ray Tune (turn on by setting `ray_tune_reporter`)
```

Comment:
What happens if `ray_tune_interval < 0` (the default) and `ray_tune_reporter` is not None? Should we be checking the interval is positive? (I think `x % 0` will also cause issues.)

Reply:
I decided to rely on Stable Baselines for now, so these docs/code have been deleted.